src/tools/cephfs/MetaTool.cc

```diff
@@ -773,10 +773,7 @@ int MetaTool::list_meta(meta_op &op)
 }
 unsigned long long MetaTool::conv2hexino(const char* ino)
 {
-  unsigned long long iino = 0;
-  std::stringstream conv;
-  conv << ino;
-  conv >> iino;
+  unsigned long long iino = std::stoull(ino, NULL, 0);
```

Comment:
could you drop `MetaTool::conv2hexino()` ? see:

```
git grep -w conv2hexino                                                                                                                                    
src/tools/cephfs/MetaTool.cc:    inodeno_t i_ino = conv2hexino(ino.c_str());
src/tools/cephfs/MetaTool.cc:    inodeno_t i_ino = conv2hexino(ino.c_str());
src/tools/cephfs/MetaTool.cc:    inodeno_t i_ino = conv2hexino(ino.c_str());
src/tools/cephfs/MetaTool.cc:    inodeno_t i_ino = conv2hexino(ino.c_str());
```

i don't see the point having it.